setup/postgresql_bootstrap_centos7.sh

```diff
@@ -88,6 +88,9 @@ function configure_postgresql () {
   # postgres performance issues under heavy load and with postgres maintenance tasks (ex. autovacuum).
   sudo sh -c 'echo "never" > /sys/kernel/mm/transparent_hugepage/defrag'
   sudo sh -c "echo 'echo \"never\" > /sys/kernel/mm/transparent_hugepage/defrag' >> /etc/rc.d/rc.local"
+  
+  # Kernel migration cost
+  sudo sh -c 'echo "5000000" > /proc/sys/kernel/sched_migration_cost_ns'
```

Comment:
Looks like the params name changed from kernel.sched_migration_cost to kernel.sched_migration_cost_ns in some kernel version. We might have to test for one or the other before setting it

Reply:
Centos7 kernel?

Reply:
ok so stick with sched_migration_cost_ns.
However you'll also need to add `sudo sh -c "echo 'echo \"5000000\" > /proc/sys/kernel/sched_migration_cost_ns' >> /etc/rc.d/rc.local"`
otherwise the setting will disappear next reboot